test/Java9andUp/build.xml

```diff
@@ -61,8 +62,9 @@
 				<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
 					<src path="${src}"/>
 					<src path="${src_current}"/>
+					<src path="${TestUtilities}" />
```

Comment:
Does it make sense to just include the attachAPI related java files? TestUtilities will have utils for other projects in the future, so it is unnecessary to build entire TestUtilities project.

Reply:
Given that TestUtilities is small at present and should not grow quickly (compared to the test directories), I suggest including the whole thing in the interest of simplicity. We can refactor later if it becomes an issue.

Reply:
Personally, I think if we include the whole project, it will be harder for us to identify the reference between test projects and TestUtilities during refactoring time. However, if including attachAPI related java files will make the code too complicated, then I am ok with the change.

Reply:
Lan
 I’d like to hear more about how you would like to see the directory structured.
Have multiple source directories (e.g. AttachApi, StackWalker) in the TestUtilities project and add them selectively to the test projects (e.g. Java9AndUp)?  I could go for that.

Reply:
To clarify, TestUtilities is only for utility classes used by multiple test projects.  Utilities used be a single test project should live in that project.

Reply:
Another option is TestUtilities could have its own build.xml, refer to https://github.com/eclipse/openj9/issues/170 .